recipe/meta.yaml

```
@@ -20,22 +20,22 @@ outputs:
       run_exports:
         strong:
           - cudatoolkit {{ cuda_compiler_version }}|{{ cuda_compiler_version }}.*
-          - __glibc >=2.17  # [linux and cuda_compiler_version == "11.0"]
+          - __glibc >=2.17  # [linux and cuda_compiler_version in ("11.0", "11.1")]
```

Can we use string comparisons so we don't have to keep adding here whenever a new CUDA version is released?
```suggestion
          - __glibc >=2.17  # [linux and cuda_compiler_version >= "11.0"]
```

---

I tried with int-tuples but the selector syntax does not work well with them. See previous commits and maybe you'll see something I missed?

---

Hm, I figured out a way to do it but it's _not_ pretty. See commit 7ca2d6a.

---

Do we need to convert the major version to an int? I was suggesting using string comparison, but that does fail if/when there's a CUDA version 100 😄.

---

9.2 would be greater than 11 if we use string comparison:

```
>>> "9.2" > "11.0"
True
```

---

Resolving this, forgot we're still supporting 9.2 for now.